Jenkinsfile

```diff
@@ -0,0 +1,115 @@
+                    inheritFrom 'default'
+                    yamlFile './cbci-templates/fmforci.yaml'
+                }
+            }
+
+            when {
+                branch 'master'
+            }
+
+            steps {
+                container("rox-proxy") {
+                    waitForRoxProxy()
+                }
+
+                container(name: "server", shell: 'sh') {
+                    withCredentials([
+                        string(credentialsId: 'TEST_E2E_BEARER', variable: 'TEST_E2E_BEARER'),
+                        sshUserPrivateKey(credentialsId: 'SDK_E2E_SSH_KEY', keyFileVariable: 'SDK_E2E_SSH_KEY', passphraseVariable: '', usernameVariable: 'cloudbees.eslint@cloudbees.com'),
+                        sshUserPrivateKey(credentialsId: 'SDK_E2E_TESTS_DEPLOY_KEY', keyFileVariable: 'SDK_E2E_TESTS_DEPLOY_KEY', passphraseVariable: '', usernameVariable: 'cloudbees.eslint@cloudbees.com'),                        
+                    ]) {
+                        script {
+                            addGitHubFingerprint()
+                            TESTENVPARAMS = "QA_E2E_BEARER=$TEST_E2E_BEARER API_HOST=https://api.test.rollout.io CD_API_ENDPOINT=https://api.test.rollout.io/device/get_configuration CD_S3_ENDPOINT=https://rox-conf.test.rollout.io/ SS_API_ENDPOINT=https://api.test.rollout.io/device/update_state_store/ SS_S3_ENDPOINT=https://rox-state.test.rollout.io/ CLIENT_DATA_CACHE_KEY=client_data ANALYTICS_ENDPOINT=https://analytic.test.rollout.io/ NOTIFICATIONS_ENDPOINT=https://push.test.rollout.io/sse"
+
+                            withEnv(["GIT_SSH_COMMAND=ssh -i ${SDK_E2E_SSH_KEY}", "PATH+GO=$PATH:/usr/local/go/bin"]) {
+                                echo "Executing E2E tests"
+                                sh script: """
+                                    apt-get update && apt-get install -y curl gnupg
+                                    curl -sL https://deb.nodesource.com/setup_lts.x | bash - 
+                                    apt-get install -y nodejs && npm install -g yarn
+                                    
+                                    git clone git@github.com:rollout/sdk-end-2-end-tests.git
+                                    ls -la
+                                    ln -s ./v6/driver ./sdk-end-2-end-tests/drivers/go
+                                    cd sdk-end-2-end-tests
+                                    yarn install --frozen-lockfile
+                                    SDK_LANG=go ${TESTENVPARAMS} NODE_ENV=container yarn test:env
+                                """, label: "Pull SDK end2 tests repository"
+                            }// end withEnv
+                        }
+                    }
+                }
+            }
+            post{
+                success{
+                    script {
+                        echo 'E2E Tests OK; posting results'
+                        currentBuild.result = 'SUCCESS'
```

Comment:
I'm not sure, but is this necessary? Surely at this stage the build result is already set correctly by definition, because we're in the `success` block rather than `failure`.
In fact, I can't even see the value referenced anywhere, is it even used?